pinot-controller/src/main/java/org/apache/pinot/controller/tuner/TableConfigTunerRegistry.java

```diff
@@ -67,11 +90,15 @@ private TableConfigTunerRegistry() {
         }
       }
     });
-    LOGGER.info("Initialized TableConfigTunerRegistry with {} tuners: {}", _configTunerMap.size(),
-        _configTunerMap.keySet());
+
+    _init = true;
+    LOGGER.info("Initialized TableConfigTunerRegistry with {} tuners: {} in {} ms", _configTunerMap.size(),
+        _configTunerMap.keySet(), (System.currentTimeMillis() - startTime));
   }
 
   public static TableConfigTuner getTuner(String name) {
+    TableConfigTunerRegistry.init(Arrays.asList("org.apache.pinot"));
```

Comment:
Remove

Reply:
Ah, good catch. Was supposed to go into the test.